WBooks/Library/LibraryViewController.swift

```diff
@@ -15,6 +15,9 @@ class LibraryViewController: UIViewController {
     // MARK: Properties
     private let libraryView: LibraryView = LibraryView.loadFromNib()!
     private let viewModel: LibraryViewModel
+    private lazy var searchBar: UISearchBar = WBooksSearchBar(containerView: view)
+    private var searchActive: Bool = false
+    private var filteredBooks = [Book]()
```

Comment:
This should be on the view model and the logic to filter it

Reply:
Done